environment-py3.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - cachetools
   - certifi
   - cmreshandler >1.0.0b4
-  - docutils
+  - docutils ~=0.17 # Bug between sphynx and docutils 0.18 https://sourceforge.net/p/docutils/bugs/431/
```

Comment:
This should be fixed by requiring `sphinx >=4.2.0` rather than pinning `docutils` (the issue is that a newer `docutils` is being preffered instead of a newer `sphinx` and the older `sphinx` versions don't have an upper limit on `docutils`).

In fact I'm pretty sure DIRAC doesn't even use `docutils` directly so it shouldn't even be here:
```suggestion
```

Reply:
It might be better to use `>=4.1.0` so there is still Python 2 support

Reply:
let's try like this